G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/ExcelRemoveRow/ShouldRemoveRow_WhenRemoveRowCommandIsCalled.G1ANT

```diff
@@ -0,0 +1,15 @@
+﻿excel.open
+
+excel.setvalue value ‴1‴ row ‴1‴ colindex ‴1‴
+excel.setvalue value ‴2‴ row ‴2‴ colindex ‴1‴
+excel.setvalue value ‴3‴ row ‴3‴ colindex ‴1‴
+
+excel.removerow row 1
```

Comment:
I would remove all unneeded ‴‴ and first argument names just like here you did

Reply:
done